frontend/src/components/tablestates.js

```diff
@@ -15,58 +15,57 @@ import {
   SearchIcon
 } from '@patternfly/react-icons';
 
-export class TableEmptyState extends React.Component {
-  static propTypes = {
-    onClearFilters: PropTypes.func,
-  }
+// TODO this could be one component with variable icons and textual content.
 
-  render() {
-    return (
-      <Bullseye>
-        <EmptyState>
-          <EmptyStateHeader titleText="No results found" icon={<EmptyStateIcon icon={SearchIcon} />} headingLevel="h5" />
-          <EmptyStateFooter>
-            {!!this.props.onClearFilters &&
-            <React.Fragment>
-              <EmptyStateBody>
-                No results match this filter criteria. Clear all filters to show results.
-              </EmptyStateBody>
-              <EmptyStateActions>
-                <Button variant="link" onClick={this.props.onClearFilters}>Clear all filters</Button>
-              </EmptyStateActions>
-            </React.Fragment>
-            }
-          </EmptyStateFooter>
-        </EmptyState>
-      </Bullseye>
-    );
-  }
+const TableEmptyState = (props) => {
+  const {onClearFilters} = props;
+  <Bullseye>
+    <EmptyState>
+      <EmptyStateHeader titleText="No results found" icon={<EmptyStateIcon icon={SearchIcon} />} headingLevel="h5" />
+      <EmptyStateFooter>
+        {!!this.props.onClearFilters &&
+        <React.Fragment>
+          <EmptyStateBody>
+            No results match this filter criteria. Clear all filters to show results.
+          </EmptyStateBody>
+          <EmptyStateActions>
+            <Button variant="link" onClick={onClearFilters}>Clear all filters</Button>
+          </EmptyStateActions>
+        </React.Fragment>
+        }
+      </EmptyStateFooter>
+    </EmptyState>
+  </Bullseye>
 }
 
-export class TableErrorState extends React.Component {
-  static propTypes = {
-    onClearFilters: PropTypes.func,
-  }
+TableEmptyState.propTypes = {
+  onClearFilters: PropTypes.func,
+}
+
+const TableErrorState = (props) => {
```

Comment:
since you are using `{}` we need to have `return`

Reply:
ah I wasn't unpacking the props before, added the `const` line and didn't add the return. thanks!